scripts/troubleshoot/LogCollection/AgentLogCollection.sh

```diff
@@ -1,36 +1,35 @@
-#!/bin/bash
-#
+# !/bin/bash
 # Copyright (c) Microsoft Corporation.
```

Comment:
@bdegs , script should be used in the ci_prod branch not the ci_dev - https://github.com/microsoft/Docker-Provider/blob/ci_prod/scripts/troubleshoot/LogCollection/AgentLogCollection.sh.   
which  link you are using which taking ci_dev branch? 
cc: @pfrcks

Reply:
@ganga1980 , used incorrect link when creating this pull request (must have been link saved by browser cache. I'll cancel this one and make one in the correct link. Thanks for the review!